ui/src/App.css

```diff
@@ -211,10 +211,10 @@
   transition: 0.3s;
 }
 
-.video-card:hover {
+/* .video-card:hover {
```

Comment:
Leftover ?

Reply:
I commented it out on purpose because it was breaking the ui, so just for the demo it should stay like this.

Reply:
Imo our code will stay in this branch so we can remove this

Reply:
Ok, deleting